src/screens/Register/Register.tsx

```diff
@@ -86,9 +86,14 @@ export const Register = () => {
                 </Typography>
 
                 <form
-                    onSubmit={handleSubmit((data) =>
-                        registerHook
-                            .mutateAsync(data)
+                    onSubmit={handleSubmit((data) => {
+                        const birthdate = new Date(data.birthdate);
+                        birthdate.setDate(birthdate.getDate() + 1);
```

Comment:
Very sus
What is your solution? How it works?

Reply:
We can just leave this solution for now, if we don't get better merge this before demo

Reply:
Ok let's leave it for now. If i don't come up with a better solution we will merge it just yours before demo